build.clj

```diff
@@ -0,0 +1,86 @@
+(ns build
+  (:require
+    [borkdude.gh-release-artifact :as gh]
+    [clojure.tools.build.api :as b]
+    [deps-deploy.deps-deploy :as dd]))
+
+
+(def lib 'replikativ/datahike)
```

Comment:
that's `io.replikativ/datahike`, have a look [here](https://clojars.org/io.replikativ/datahike)

Reply:
oh man, you're totally right, :+1: